docs/guides/preinstall.md

```diff
@@ -13,68 +23,109 @@ In macOS Disk Utility, [create a partition](https://support.apple.com/guide/disk
+
+Please, refer to this guide: [Basic setup](https://wiki.t2linux.org/guides/postinstall/).
+
+You can also find detailed info for unsupported debian-based distros [here](https://wiki.t2linux.org/distributions/debian/installation/).
 
 ### Copying the ISO to the USB
 
-User friendly applications for copying the Linux install image to your USB include:
+You can download user-friendly applications for copying the ISO to your USB:
 
 - [USBImager](https://gitlab.com/bztsrc/usbimager/)
-- [balenaEtcher](https://www.balena.io/etcher/) (does collect analytics data).
+- [balenaEtcher](https://www.balena.io/etcher/) (does collect analytics data)
+
+Or you can use the macOS terminal:
 
-!!! hint "`dd`"
-    If you are familiar with the `dd` tool and the command line, you can use that instead. Use `diskutil list` to find the correct disk, then `sudo dd if=path/to/filename.iso of=/dev/rdiskX bs=1m`. Press `control-T` while it is running to make it show its current progress. **Make sure you select the correct disks.**
+1. Insert your USB drive.
+2. Open macOS Terminal.
+3. Run `diskutil list` to list all the drives.
+4. Look up for your USB thumb which appears as an **external, physical** drive labelled `/dev/diskX` where `X` is a single number (e.g. /dev/disk**2**).
+5. Run `sudo diskutil unmountDisk /dev/diskX` to unmount the disk.
+6. Run `sudo dd if=path/to/linux.iso of=/dev/rdiskX bs=1m` to start writing. To fetch the iso path easily you can just drag and drop the .iso into the Terminal.
+7. Now you may press `control-T` to make it show how many KBs it has written so far.
 
-While the installation image is being written to the USB, you can [copy Wi-Fi firmware](#copy-wi-fi-firmware), but don't follow the steps after that until it has finished writing the image.
+While the installation image is being written to the USB, you can skip to [Copy Wi-Fi firmware](#copy-wi-fi-firmware); but don't follow the steps after it, wait until the ISO has been written to disk.
 
 ## Copy Wi-Fi firmware
 
-Linux's Wi-Fi driver uses the same Wi-Fi firmware files as macOS, so we copy these files from macOS to the EFI partition where Linux can access these files and then install them.
+Linux's Wi-Fi driver uses the same Wi-Fi firmware files as macOS, thereby we copy them from macOS to the EFI partition where Linux can access and install them.
+
+[Follow here](https://wiki.t2linux.org/guides/wifi-bluetooth/#on-macos) the instructions for the **first part done in macOS** and come back to this page.
 
-Follow the instructions in this section to be followed on macOS [here](https://wiki.t2linux.org/guides/wifi-bluetooth/#on-macos). The sections of that page to be followed on Linux can be followed later, once you've successfully installed Linux.
+The **second part "On Linux"** must be followed **after** you have completed the Linux installation.
 
 ## Disable Secure Boot
 
-!!! note
```

Comment:
Except for the steps written down to change secure boot state, the other minor edits are uncessary. Also, you’ve removed the warning notes, why?

Reply:
The warning note for "wait to finish creating your install usb" is already mentioned on line 67. I think it's redundant here after two steps.
The warning for "and you have copied Wi-Fi firmware" I also think it's redundant. Since it is already specified to come back on this guide again after finishing (line 73). 
That's why the !note has been removed in the commit.